.github/workflows/nvm.yml

```
@@ -5,9 +5,47 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: node --version
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - run: |
           nvm install --latest-npm --no-progress
-          echo "::set-env name=PATH::${NVM_BIN}:${PATH}"
+          echo "$(dirname $(which node))" >> $GITHUB_PATH
```

[suggest] Let's use quotes for environment variables.

```suggestion
          echo "$(dirname $(which node))" >> "$GITHUB_PATH"
```

---

Yeah normally I would do this but in the docs for GitHub's "magic" environment variables, they don't include quotes

---

> they don't include quotes

I didn't notice that, for example: https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action#creating-an-action-metadata-file

Okay, let's keep them as-is. 👍🏼